clients/launcher/lib/pages/settings_page.dart

```diff
@@ -1,21 +1,53 @@
+import 'dart:io';
 import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
+import 'package:launcher/env.dart';
+import 'package:launcher/widgets/welcome_modal.dart';
 import 'package:sail_ui/sail_ui.dart';
 
 @RoutePage()
 class SettingsPage extends StatelessWidget {
   const SettingsPage({super.key});
 
+  Future<void> _openDataDirectory() async {
+    final appDir = await Environment.datadir();
+    final command = Platform.isMacOS 
+      ? 'open' 
+      : Platform.isWindows 
+        ? 'explorer'
+        : 'xdg-open';
+    
+    await Process.run(command, [appDir.path]);
+  }
+
```

Comment:
If you rebase on master you can delete this and just call `openDataDirectory(appDir)` instead